### ISSUE TITLE
Burnout 3 - Embedded Video and menu high latency

### ISSUE THREAD

**Issue by jcduss59** [xemu-project/xemu#1266]
### Title

https://xemu.app/titles/4541005b/#Burnout-3-Takedown

### Bug Description

Menu and tutorial video are really slow.
In fact, when you watch the firsts tutorial videos, the audio played is ending at least 30s before the end of the videos.
Even country map is slow to display when you choose your race.

In game is working fine.

### Expected Behavior

Normal and synchronised display of menu and sound.

### xemu Version

0.6.2-80

(batocera embedded)

### System Information

OS : Batocera v34 (Arch Linux Emulstation)
CPU : Intel i7 4790
GPU : Nvidia T600 (quadro)

### Additional Context

Thank you.

**Comment by AdadG** [comment]
That's because of your processor, xemu is very CPU-bound. In my case, with a 5600X (+200MHz), the tutorial video plays almost perfectly, more than 95% of the time at 59 fps (the most common highest rate), without synchronization problems, the other videos are worse but also have no synchronization problem. The menu is very erratic but it doesn't interfere with gameplay or spoil the experience. So, is your hardware, not the emulator, therefore I think you should close this issue request.

**Comment by ghost** [comment]
I have the same problem, videos, menus and tutorials are slow but the game play is fast and plays great. No other games do this at this point, here's my specs:

System Information:

Windows 11
i9 9980HK 8 core 16 Thread
RTX 2080 Max Q 8GB
64 GB memory

**Comment by jcduss59** [comment]
I dont think it comes from my hardware. When i display the head up diag, in menu or in game I reach 60fps. Not lost frame rate.

And the sound is not laggy. While @MachWon bas the same issue with stronger hardware than mine.

**Comment by AdadG** [comment]
Well, that's very rare then, I mean the 30 second desync, that's too much, especially if the gameplay is going well for you like you said, have you checked your dump file?
For the rest, as you can see on Discord, the Crash Nav or menu is a well known issue.
In short, such problems do exist but, on certain hardware, they shouldn't be as pronounced as you've mentioned.

**Comment by AdadG** [comment]
Yes indeed the menu issue is a well known one as you can see on Discord but now it seems more strange to me that you say that in game you reach 60fps with no frame rate loss (which is already hard) but in the initial video you have a desync of 30 seconds, that time is too long, have you checked your dumped file? That or there is no direct relationship between both things.

**Comment by jcduss59** [comment]
You mean this is a known issue on discord ( i dont use discorde btw). But I should check my dump ROM?
I play with an EU PAL dump. Do you know if others on discord share the same one?

A suggestion would be to share the checksum (md5 or sha1) of each tested games on the compatibility page. I should help a lot. 

I'll extract mine.

**Comment by jcduss59** [comment]
A point that I dont get too is that in game it's strong 3D to display (many ressources needed) while to display the map of challenge (Europe, US..). There is nothing special to display.

**Comment by AdadG** [comment]
Oh that's the cause of the confusion, I assumed that you were in Discord. I highly recommend to you using it and joining to the xemu's channel, so you will be aware, you can get and give support, it's also a good community. In fact, curiosly, in a recent comment I suggested that in cases like this a warning to verify a potential problem with someone else before publishing it would be good.

I have the same perplexity, I don't know what problem is there.

**Comment by jcduss59** [comment]
Here is the sha1 
f03cd17b42104042e408e811cb9fd0f15afd5449  Burnout 3 - Takedown (Europe) (Fr,De,It).iso

I am pretty sure It's a "redump" iso" and I had the "please insert an xbox image" issue. So I "dd" it with following parameters to make it works : skip=387 BS=1M 
and then i was starting fine.

**Comment by jcduss59** [comment]
what could i verify on my iso to validate it's correct?

**Comment by SourceCodeDeleted** [comment]
I have the same issue. I was just hoping for something to make the menus faster.

Other than that, the game runs fine . So it's not the end of the world.

**Comment by Rastafabisch** [comment]
Sadly still very much true to this day. Are there any debugging steps I could try to get to the root cause of that issue?

|Rendered/3D content, high fps|Menu, low fps|
|---|---|
| <img width="2400" alt="Burnout 3 Takedown, 3D" src="https://github.com/user-attachments/assets/c4d07104-2e4d-46a3-81a3-f610d19a28a6" /> |  <img width="2402" alt="Burnout 3 Takedown, Menu" src="https://github.com/user-attachments/assets/b643e24a-34bd-4b97-b69a-9e075ca724d0" /> |

**Comment by SourceCodeDeleted** [comment]
I just decided to live with it. The MP split screen doesn't work either.
